internal/v2/domain/application.go

```diff
@@ -11,7 +11,7 @@ type Application struct {
 	State      AppState
 	Name       string
```

Comment:
now that we have OIDCApp we should remove State und Name

Reply:
we still need it to change application